packages/react-vapor/src/components/blankSlate/BlankSlateWithError.tsx

```diff
@@ -0,0 +1,33 @@
+import * as VaporSVG from 'coveo-styleguide';
+import * as React from 'react';
+import {Svg} from '../svg/Svg';
+import {IBlankSlateProps} from './BlankSlate';
+
+export const blankSlateWithError = () => (
```

Comment:
```suggestion
export const blankSlateWithError = (
```
That way, you wouldn't have to call the function
```ts
export const BlankSlateWithTableInError = _.compose(
    blankSlateWithTable(),
    blankSlateWithError,
)(BlankSlate);
```